Sources/androidNativeInit.swift

```diff
@@ -10,13 +10,13 @@ import SDL
 import CJNI
 
 public struct UIKitAndroid {
-    @_silgen_name("SDL_Android_Init")
+    @_cdecl("SDL_Android_Init")
```

Comment:
for some reason _cdecl can only be used with top level functions

Reply:
This is more puzzling though. It shouldn’t make any difference whether it’s top-level or static from what I understand. I’d quite like to define these as cdecl because otherwise we may end up with unexpected behaviour again. So we should take them out of the struct and back to top level